examples/sampleProp.conf

```diff
@@ -3,6 +3,7 @@ GatewayName = Gateway-Name
 MetricName = Metric-Name
 MetricWithOwnTsName = Metric-With-Own-Ts-Name
 BulkCollectedMetricName = Bulk-Collected-Metric-Name
+DeviceName = Gateway-Name
```

Comment:
Can you move this up the order. Generally one creates a Gateway, a Device and then a Metric. Also, change its default value to Device-Name.